content/_index.md

```diff
@@ -26,6 +26,11 @@ features:
     image_path: /images/icon-multilingual2.svg
     tagline: Polyglot baked in.
     copy: Hugo provides full i18n support for multi-language sites with the same straightforward development experience Hugo users love in single-language sites.
+
+  - heading: Custom Outputs
+    image_path: /images/icon-custom-outputs.svg
+    tagline: HTML not Enough?
+    copy: Hugo allows you to output your content in multiple types, like JSON, AMP, or you can create your own.
```

Comment:
It's not a huge distinction, but this was something @bep had called out in a previous conversation since Hugo doesn't deal with images, video, etc. I was offering two alternatives, but I think the copy I included in the blockquote above is the best of the three. The copy edit is more a matter of parallel structure. 

And oops, you can ignore the PR I submitted on the theme. Sorry I missed that in the above comment 🙇